snackager/package.json

```diff
@@ -37,7 +37,7 @@
     "semver-utils": "^1.1.4",
     "snack-content": "*",
     "snack-require-context": "*",
-    "snack-sdk": "*",
+    "snack-sdk": "4.1.0-rc.1",
```

Comment:
question: do you know for a reason to change this? would that help monorepo managing? if somehow we bump publish newer snack-sdk, should we change this manually? or turborepo could help that?

Reply:
I guess `yarn@1` has a lot of issues connecting `"*"` versions to `-alpha|beta|rc.x`. I'll move this change from the PR to the `sdk-49` upgrade and resolve the e2e failures.

I also want to split up some workflow changes from this PR into their own.

Reply:
I had a similar issue with the website: https://github.com/expo/snack/pull/437

Reply:
that makes sense. pre-releases always act differently. that's by design of semver. i had a trick in [install-expo-modules](https://github.com/expo/expo-cli/blob/5bac95d361227497e53c99f3606294546d872040/packages/install-expo-modules/src/utils/packageInstaller.ts#L32-L37), but i don't know how to use wildcard to include a pre-release version for the case here.

maybe an alternative solution here is like publishing without pre-release (e.g. snack-sdk 4.2.0) and with `next` dist-tag.

Reply:
Yeah, that seems like a good alternative. I'd love if we can swap to some tool that allows us to be explicit in workspace versions, e.g. `"snack-sdk": "workspace:*"` (whatever is available in the workspace).

Reply:
> I'd love if we can swap to some tool that allows us to be explicit in workspace versions, e.g. "snack-sdk": "workspace:*" (whatever is available in the workspace).

that's a neat idea. hopes there are some package manager to support this semantic.

Reply:
Currently support is:

- `yarn@>=2` - [docs](https://yarnpkg.com/features/workspaces#cross-references)
- `pnpm` - [docs](https://pnpm.io/workspaces#workspace-protocol-workspace)
- `bun` - [docs](https://bun.sh/docs/install/workspaces)
- `npm` - no support, no RFC open (yet... 😄)

Reply:
good to know the workspace protocol was in real. thanks Cedric!